.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 # Keep environment variables out of version control
 .env
+prisma
```

Comment:
I don't think excluding the whole prisma folder is a good idea. That incudes the schema which drives the database.

Reply:
I thought we have removed it by resolving the conflict last time, but request on changes has been approved